.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["amatiasq.sort-imports"]
```

Comment:
Well I'm not sure if you noticed my message on Discord but this is what I use currently since it does exports as well, and respects ESLint rules. Plus removes unnecessary imports.

https://marketplace.visualstudio.com/items?itemName=dozerg.tsimportsorter#vscode-settings

Reply:
Ignore this, I'll remove it. I'm using the one you have recommended. Which files have wrong import order?

Reply:
Updated the recommended extensions, take a look when you have time. Also I could not find any file with problems of sort importing. I'll try to find a prettier rule for that and maybe we can fix all imports with one command

Reply:
I guess we have already a rule on eslint for sort imports, but it conflicts with the extension on vscode. So if run yarn lint-fix, it will sort the imports. If I save the file, the imports will be sorted by the extension, with different criteria. Maybe it is better to not use the extension and do a lint-fix before pushing the commit.

Reply:
Weird. I'll have a look into it later. Drop the extension for now I guess. Let's get this merged.